torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3443,16 +3444,16 @@ def sample_inputs_instance_norm(opinfo, device, dtype, requires_grad, **kwargs):
     def generator():
         for input_shape, kwargs in cases:
             # args: running mean, running var, weight and bias should necessarily be of shape: (channels,)
-            channels = input_shape[1] if len(input_shape) > 1 else 0
-            weight = make_arg(channels) if channels > 0 else None
-            bias = make_arg(channels) if channels > 0 else None
+            channels = input_shape[1]
+            weight = make_arg(channels)
+            bias = make_arg(channels)
```

Comment:
The None option isn't needed anymore?

Reply:
Yeah this was some weird logic that had gotten in there from some back and forth refactoring--since we add the inputs that try instance_norm with None in the secondary case below so that we can try permutations of weights and bias being None, the None case never shows up in this part